rqd/rqd/cuerqd.py

```diff
@@ -13,38 +13,11 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+#--------------------------------------------------------------------------
 
-
-"""SYNOPSIS
-     cuerqd [hostname] [OPTIONS]
-      [hostname]            => RQD hostname (defaults to localhost)
-      -s                    => Print rqd status
-      -v                    => Print rqd version
-      --lp <cores>          => Lock Cores
-      --ulp <cores>         => Unlock Cores
-      --lh                  => Lock Host (All Cores)
-      --ulh                 => Unlock Host (All Cores)
-      --nimbyoff            => Turn on Nimby
-      --nimbyon             => Turn off Nimby
-      --exit                => Lock host, wait until machine is idle and then Shutdown RQD *
-      --exit_now            => KILL ALL running frames and Shutdown RQD
-      --restart             => Lock host, wait until machine is idle and then Restart RQD *
-      --restart_now         => KILL ALL running frames and Restart RQD
-      --reboot              => Lock host, wait until machine is idle and then REBOOT machine *
-      --reboot_now          => KILL ALL running frames and REBOOT machine
-    print
-      --kill <frameid>      => Attempts to kill the given frame via its ICE proxy
-      --getproxy <frameid>  => Returns the proxy for the given frameid (debug)
-    print
-     * Any unlock command will cancel this request
-\n FOR TESTING:
-      --test_edu_frame        => Launch a test edu frame on an idle core
-                                Use first core if none are available
-      --test_script_frame     => Same as above but launches a 5 second python script
-      --test_script_frame_mac => Same as above but for mac host
-\nDESCRIPTION
-      Displays information from or sends a command to an RQD host"""
-
+""" Help
+       Use following at command line : cuerqd [-h or --help]
```

Comment:
This doesn't look quite right to me, as I'm assuming (I haven't tried running the command) that this is the string that is displayed when the user runs `cuerqd -h`. If so, then we really want this string to explain what `cuerqd` does?

If so, the original description would be OK:

Displays information from, or sends a command to, an RQD host.

If this isn't the right place to set the description, we should make sure we migrate the description as I can't see it in the updated code.

Reply:
This message is not displayed with cuerqd -h.